src/App.css

```diff
@@ -18,3 +37,207 @@ nav,
   column-gap: 24px;
   align-items: center;
 }
+
+.links a {
+  opacity: 0.5;
+}
+
+.links a.active {
+  opacity: 1;
+}
+
+.login {
+  border: 0.1px solid #dcd7d7;
+  border-radius: 50%;
+  padding: 0.875rem;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--azure);
+  margin-left: auto;
+  cursor: pointer;
+}
+
+/* booklist */
+.book-list {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin-bottom: 2.5rem;
+}
+
+.book {
+  min-height: 10.625rem;
+  padding: 2rem 1.688rem;
+  background-color: #fff;
+  border-radius: 4px;
+  border: solid 1px var(--grey);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  margin-bottom: 0.25rem;
+}
+
+.title {
+  font-size: 1.375rem;
+  font-weight: 700;
+  letter-spacing: -0.2px;
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
- [ ] (Optional) Since you used this `font-family` consistently in your code, it would be nice to make a CSS variable and use it when needed. You can learn more about CSS variables by reading this [article](https://www.w3schools.com/css/css3_variables.asp). 👍